src/aics-image-viewer/components/useVolume.ts

```diff
@@ -175,6 +175,11 @@ const useVolume = (
   const [channelGroupedByType, setChannelGroupedByType] = useState<ChannelGrouping>({});
 
   const onChannelDataLoaded = useCallback(
+    // TODO FIXME this can be called immediately during volume creation, before
+    // the channel settings are set up.  This can happen during RawArrayLoader usage.
+    // The Raw Array Loader does not go across thread boundaries and it doesn't ahve to do
+    // anything asynchronous, so this just gets called immediately.
+    // But the onChannelLoadedRef callback is going to index into the channel state array, which is not set up yet.
```

Comment:
@frasercl I could use a consult with you on how best to handle this.  In my testing, wrapping the body of this function below in a setTimeout(..., 0) was sufficient to get the raw array to load without error.